titus-common/src/main/java/com/netflix/titus/common/runtime/SystemLogService.java

```diff
@@ -0,0 +1,24 @@
+package com.netflix.titus.common.runtime;
+
+/**
+ * An API for logging critical system events (errors, exceptions, invariant violations) with supplementary metadata information.
+ * The events are classified by their category and priority for better searchability.
+ */
+public interface SystemLogService {
+
+    enum Priority {
```

Comment:
What does priority mean here?  Is it different than the typical `INFO` `WARN` `ERROR` `FATAL` type levels?

Reply:
I was thinking about something more actionable. For example:
* set `Critical` for a prolonged lack of connectivity to an external system
* set `Low` if the connection is dropped but we can restart quickly

However, maybe the typical levels are good enough and we should stick to them.